cmd/bots/internal/common/environment.go

```diff
@@ -615,7 +614,7 @@ func sortNodesStatuses(statuses []NodeStatus) {
 
 func executeTemplate(template string, data any, extension ExpectedExtension) (string, error) {
 	switch extension {
-	case HTML:
+	case HTML, Markdown:
 		tmpl, err := htmlTemplate.ParseFS(templateFiles, template+string(extension))
```

Comment:
Package renaming from `template` to `htmlTemplate` is not required any more.

Reply:
Fixed.